scripts/zones/Southern_San_dOria/npcs/Varchet.lua

```diff
@@ -18,7 +18,6 @@ local GAME_TIE = 3;
 -----------------------------------
 
 function onTrade(player,npc,trade)
-
     if (npcUtil.tradeHas(trade, nil, 5)) then
```

Comment:
Should I perhaps switch this to trade:getGil? Or does it not really matter? It works just fine as it is, but if tradeHas is deprecated for gil, it might be easier to read if it's :

```
if (trade:getGil() == 5) then
```